lib/kafka_ex/config.ex

```diff
@@ -0,0 +1,17 @@
+defmodule KafkaEx.Config do
+  @moduledoc false
+
+  def default_worker do
+    :kafka_ex
+  end
+
+  def server_impl do
+    :kafka_ex
+      |> Application.get_env(:kafka_version, :default)
+      |> server
+  end
+
+  defp server("0.8.0"), do: KafkaEx.Server0P8P0
```

Comment:
Not sure this belongs here, it's not really "configuration".  Not a huge deal, though.